src/ucs/sys/sys.c

```diff
@@ -1354,6 +1354,11 @@ int ucs_sys_getaffinity(ucs_sys_cpuset_t *cpuset)
     return ret;
 }
 
+int ucs_sys_pthread_getaffinity(pthread_t tid, ucs_sys_cpuset_t *cpuset)
```

Comment:
Pass `tid`, then call `pthread_self()`? Why?

Reply:
leftovers... no need to pass it of course